Gemfile

```diff
@@ -23,7 +24,14 @@ group :development, :test do
   gem 'puppetlabs_spec_helper',  :require => false
   gem 'puppet-lint',             :require => false
   gem 'simplecov',               :require => false
-  gem 'beaker-rspec', *location_for(beaker_rspec_version)
+  if beaker_version
+    gem 'beaker', *location_for(beaker_version)
+  end
+  if beaker_rspec_version
+    gem 'beaker-rspec', *location_for(beaker_rspec_version)
+  else
+    gem 'beaker-rspec',  :require => false
+  end
```

Comment:
Would you prefer to simply remove the beaker/beaker-rspec gem dependency until such time as there are acceptance level tests?  If so, I'm cool with that.  I was just folding the Gemfile under the new standard without really being concerned with the actual contents of that file.